### DISCUSSION TITLE
Format users field correct, from page blueprint?

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [johannschopplich/nuxt-kql#20] [Q&A]
Hello,
I have added a users field to the page blueprint.
But when I bring it into nuxt via a query. it comes in as

`{ blocks:
   [ { content: [Object],
       id: '7d17fe25-3d46-4352-ab05-ec3037aefe0e',
       isHidden: false,
       type: 'text' } ],
  uri: 'blog/test2',
  title: 'test2',
  date: '2024-09-19',
  updated: '2024-09-30',
  author: "- 'user://rSGuMHwF'",
  intendedTemplate: 'article',
  description: '',
  cover: null }`

see the authors property, that is the kirby users field.

Is there a correct way to format this like an array such as in the notes header block?

**Comment by johannschopplich** [comment]
This is default behaviour from KQL. If you want to resolve UUIDs to model objects fields inside blocks, such as a`files` field, make sure to follow the [`toResolvedBlocks()` field method guide](https://kirby.tools/docs/headless/field-methods#toresolvedblocks). This will resolve the data on the server. Use this as an alternative to `toBlocks`.

By the way, feel free to check out the [Cacao Kit](https://github.com/johannschopplich/cacao-kit-frontend/), which already covers common obstacles like this one.

**Reply by pixelpaulaus** [reply]
thanks, i am using that Kit already.

**Reply by johannschopplich** [reply]
Good to know! The [`TeamStrucuture`](https://github.com/johannschopplich/cacao-kit-frontend/blob/main/components/Kirby/Block/TeamStructure.vue) is an example block that consumes resolved data from `toResolvedBlocks`. 

In the backend, the data is resolved here for that block: 
https://github.com/johannschopplich/cacao-kit-backend/blob/df79d193a17ea7f0717aa077c4ebbbef3fe48c9e/site/config/blocks-resolver.php#L17
